<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>biblioteca</class>
 <widget class="QDialog" name="biblioteca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biblioteca</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#biblioteca {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 0, y2: 1,
        stop: 0 #3f6a8a,
        stop: 0.5 #2a2a2a
    );
}
</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>REGISTRO DE LIBROS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Agregar_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titulo_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="autor_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Título: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>319</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro Medium</family>
    </font>
   </property>
   <property name="text">
    <string>---------------------------------- L I B R O S   R E G I S T R A D O S --------------------------------</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Eliminar_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>571</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Año de publicación:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="publi_lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="actualizar_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
